tests/unit/config/argv_test.c

```diff
@@ -37,32 +37,22 @@ MTF_DEFINE_UTEST(argv_test, deserialize_to_params_invalid_param)
 MTF_DEFINE_UTEST(argv_test, deserialize_to_params_invalid_value)
 {
     merr_t             err;
-    const char * const paramv[] = { "fanout=0" };
+    const char * const paramv[] = { "prefix.length=999999999" };
```

Comment:
Might be easier to use `-1` as the invalid value.

Reply:
Will do...

Reply:
Done.